docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/how_to_connect_to_sqlite_data.md

```diff
@@ -23,33 +26,25 @@ In this guide we will demonstrate how to connect Great Expectations to data in a
 
 ### 1. Import GX and instantiate a Data Context
 
-```python title="Python code"
-import great_expectations as gx
-
-context = gx.get_context()
-```
+<ImportGxAndInstantiateADataContext />
 
 ### 2. Determine your connection string
 
 For this example we will use a connection string to connect to our PostgreSQL database.  In PostgreSQL, connection strings are formatted like:
 
-```python title="Python code"
-my_connection_string = "sqlite:///<PATH_TO_DB_FILE>"
+```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sqlite_data.py connection_string"
```

Comment:
@TrangPham Here is a small issue I also encountered.  The document says `my_connection_string = "sqlite:///<PATH_TO_DB_FILE>"`, but the code snippet says something else.  I dealt with this by redefining the `my_connection_string` variable so that creating the `Datasource` snippet stays the same.  If there is a better way, then please feel free (and let us know so we can update other tests).  Thanks.